src/test/java/co/cask/hydrator/plugin/batch/HTTPToHDFSActionTest.java

```diff
@@ -104,19 +112,20 @@ public static void setupTestClass() throws Exception {
   }
 
   @AfterClass
-  public static void teardown() {
-    httpService.stopAndWait();
+  public static void teardown() throws Exception {
+    httpService.stop();
   }
 
   @After
   public void cleanupTest() throws IOException {
     resetFeeds();
+    deleteFiles(resourceFolder);
```

Comment:
isn't this automatically done by TemporaryFolder?

Reply:
fixed.